docker.Makefile

```diff
@@ -128,22 +128,15 @@ build-e2e-image:
 	IMAGE_NAME=$(E2E_IMAGE_NAME) VERSION=$(VERSION) docker buildx bake e2e-image
 
 .PHONY: test-e2e
-test-e2e: test-e2e-non-experimental test-e2e-experimental test-e2e-connhelper-ssh ## run all e2e tests
+test-e2e: test-e2e-always-experimental test-e2e-connhelper-ssh ## run all e2e tests
 
-.PHONY: test-e2e-experimental
-test-e2e-experimental: build-e2e-image # run experimental e2e tests
+.PHONY: test-e2e-always-experimental
+test-e2e-always-experimental: build-e2e-image # run non-experimental e2e tests
```

Comment:
(See above); also no real need to mention `non-experimental` (or `experimental`) now.

```suggestion
test-e2e-local: build-e2e-image # run e2e tests
```